File: 1903-largest-odd-number-in-string/1903-largest-odd-number-in-string.java
class Solution {
    public String largestOddNumber(String num) {
        
        int length = num.length();

        for(int i = length - 1 ; i >= 0 ; i--)
        {
            int currNum = num.charAt(i) - '0';

            if(currNum % 2 == 1)
                return num.substring(0, i + 1);
        }

        return "";
    }
}

File: 1903-largest-odd-number-in-string/1903-largest-odd-number-in-string.js
/**
 * @param {string} num
 * @return {string}
 */
var largestOddNumber = function(num) {
    
    let length = num.length; 

    for(let i = length - 1 ; i >= 0 ; i--)
    {
        if(num.charAt(i) % 2 == 1)
            return num.slice(0, i + 1);
    }

    return '';
};